.travis.yml

```diff
@@ -139,6 +153,16 @@ script:
       make -j3;
     fi
 
+after_success:
+  - if [[ "$BUILD" != "Kodi" ]] && [[ "$DOCS" == "kodi-addon-dev-kit" ]]; then
+      if [[ $(git -C $TRAVIS_BUILD_DIR  config --get remote.origin.url | tr ':' '/' | awk -F ''.com/'|'.org/'' '{print $2}' | awk -F '/' '{print $1}') == AlwinEsch ]] &&
+         [[ $TRAVIS_BRANCH == next-addonlib-rework ]]; then
+        export DOC_UPLOAD_URL=esmasol.de &&
+        export DOC_UPLOAD_PATH=addon-dev-kit &&
```

Comment:
This two parts are also needed to change to a side for Kodi.

Currently my own for test.